terraform/setup/vpc.tf

```diff
@@ -10,9 +10,25 @@ resource "aws_security_group" "ec2_security_group" {
   vpc_id = data.aws_vpc.default.id
 
   egress {
-    from_port   = 0
-    to_port     = 0
-    protocol    = "-1"
+    from_port   = 443
+    to_port     = 443
+    protocol    = "TCP"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  // Allow access to IMDS
+  egress {
+    from_port = 80
+    to_port   = 80
+    protocol = "TCP"
+    cidr_blocks = ["169.254.169.254/32"]
+  }
+
+  // Allow access to EFS. https://docs.aws.amazon.com/efs/latest/ug/troubleshooting-efs-mounting.html#mount-hangs-fails-timeout
+  egress {
```

Comment:
[Its inbound right? ](https://github.com/aws/amazon-cloudwatch-agent-test/pull/132/files/56341b6fe72e027d9db48f82e815875b03ad097e..bd6485bdeab50fea3af2ae4af4bea5cc917644e5#diff-3d3018c027b1f2ed0412ba5496014b7f2ce074be95bf6899a5543ad60031c68aR61-R66)

Reply:
Inbound is already there. At the bottom of the doc, it says that if you don't allow all traffic outbound, you have to allow NFS outbound in the security group.

Reply:
Interesting. Thanks for letting me know.